go/inst/instance_topology.go

```diff
@@ -2824,6 +2826,20 @@ func relocateReplicasInternal(replicas [](*Instance), instance, other *Instance)
 			// something was moved via GTID; let's try further on
 			return relocateReplicasInternal(unmovedReplicas, instance, other)
 		}
+
+		// Making sure that if there are any errors in errs, they are reported 
```

Comment:
Please see my comment in Jira. I'm not sure if this is what was requested. This fix also changes the flow because we have 'return' here.